MessengerOnFileForLinux/FileHandling/src/FileHandling.cpp

```diff
@@ -335,10 +335,9 @@ bool FileInterface::Modification::updateRowField(const std::string& pathToFile,
     std::string row = ConsolControl::getStdoutFromStartedCommand(command.c_str());
 
     std::string rowToUpdate;
-    auto it = row.begin();
     int oldFieldSize = 0;
 
-    for (auto& x : row)
+    for (auto it = row.begin(); it != row.end();)
```

Comment:
:(

Reply:
Jeśli to ma być tak to ++it powinno być nie na końcu fora całego, a  po ; w pierwszym nawiasie. Dodatkowo powinno to być w petli zakresowej, a 338 nie powinno być. To da się zrobić.

Reply:
Najlepiej to nie edytować całej linii tylko działać na środku stringa. Ogólnie olej to, zrobimy to przy refactorze FileHandlingu

Reply:
ok